src/cgame/cg_servercmds.cpp

```diff
@@ -626,9 +626,9 @@ static void CG_Menu( int menuType, int arg )
 			break;
 
 		case MN_H_NOENERGYAMMOHERE:
-			longMsg = _("You must be near a Reactor or a powered Armoury "
+			longMsg = _("You must be near a Reactor, a Drill or a powered Armoury "
```

Comment:
this part can simply be eradicated I'd bet, because `MN_H_NOENERGYAMMOHERE` is **never** used. Well, except, of course, if some magic is done somewhere to refer to `MN_NONE + xx`, but I'd dare say this would be a quite serious bug.

Reply:
I think I've seen you saying that you'd like seing those messages back, so I'm not deleting them for now. Doesn't matter too much I guess